src/pq_flash_index.cpp

```diff
@@ -777,15 +769,45 @@ int PQFlashIndex<T, LabelT>::load_from_separate_paths(uint32_t num_threads, cons
 
     this->_num_points = npts_u64;
     this->_n_chunks = nchunks_u64;
+#ifdef EXEC_ENV_OLS
+    if (files.fileExists(labels_file))
```

Comment:
optional: I feel argument name "files" is a somewhat misleading name, it naturally sound like files in file system. Maybe better to use memFiles or memMappedFiles

Reply:
sounds reasonable it's not related to this PR though.